LiveLines.Lines/LinesService.cs

```diff
@@ -33,6 +36,37 @@ public async Task<Line> CreateLine(LoggedInUser loggedInUser, LineToCreate lineT
             ? await _songService.AddSong(lineToCreate.SpotifySongId)
             : null;
 
-        return await _linesStore.CreateLine(loggedInUser, lineToCreate.Body, songId);
+        return await _linesStore.CreateLine(loggedInUser, lineToCreate.Body, songId, lineToCreate.ForYesterday);
+    }
+
+    public async Task<LineOperations> GetLineOperations(LoggedInUser loggedInUser)
+    {
+        bool canPostToday = false;
+        bool canPostYesterday = false;
+
+        DateTime today = DateTime.Today;
+
+        try
+        {
+            await _linesStore.GetLineForDate(loggedInUser, today);
+        }
+        catch (LinesStoreException)
+        {
+            canPostToday = true;
+
+            if (DateTime.Now.Hour < 12)
```

Comment:
Do we need this, can't we just use DateTime.Now in the today variable as well? Should probably use DateTimeUTC to make the time zone obvious, too.